conf/evolutions_postgres/001-add-organizations.sql

```diff
@@ -0,0 +1,108 @@
+-- https://github.com/scalableminds/webknossos/pull/2427
+
+-- Note: _organizationTeam needs to be the ID of the previous root team
+
+-- UP:
+
+START TRANSACTION;
+
+CREATE TABLE webknossos.organizations(
+  _id CHAR(24) PRIMARY KEY DEFAULT '',
+  _organizationTeam CHAR(24) NOT NULL UNIQUE,
+  name VARCHAR(256) NOT NULL,
+  created TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  isDeleted BOOLEAN NOT NULL DEFAULT false
+);
+
+CREATE VIEW webknossos.organizations_ AS SELECT * FROM webknossos.organizations WHERE NOT isDeleted;
+
+INSERT INTO webknossos.organizations(_id, _organizationTeam, name)
+VALUES('5ab8b6be85000085008301c6', '5301a808ee7d7aab4053980f', 'Connectomics department');
```

Comment:
You use a couple of hard-coded values in this migration. Please document how you got these values, so we know what to do for other instances.